packages/swingset-runner/demo/megaPong/bootstrap.js

```diff
@@ -0,0 +1,32 @@
+import harden from '@agoric/harden';
+
+console.log(`=> loading bootstrap.js`);
+
+export default function setup(syscall, state, helpers) {
+  function log(what) {
```

Comment:
oh hey, we should remove this: the `helpers.log` will fill an ever-increasing array with messages (contributing to the apparent memory usage), and the `console.log` will probably `fflush()` stderr (contributing to the apparent runtime)

We use this pattern in unit tests where the driver function wants to know that some number of milestones have been reached from the vat code, and we copy those messages to `console.log` so that if something goes wrong we can tell from stderr more easily.. we need neither in a performance test.

Reply:
Although.. ok, I see that this isn't being called on every single message, so maybe it's not a big deal.

I'd still remove the `helpers.log` unless you've got code that's specifically checking it.

Reply:
I think the log function being here per se is probably OK if we can arrange to limit its actual use to testing only.